app/controllers/admin/all_galleries.py

```diff
@@ -1,14 +1,30 @@
 from . import admin
 from app.logic.validation import *
+from werkzeug.security import check_password_hash
 from app.models.Forms import *
+from flask import session
+from app.models.Galleries import *
 from flask import render_template
+from app.models.queries.GalleryQueries import *
 
 @admin.route('/', methods=["GET","POST"])
 def all_galleries():
-    user = Forms.get(fid=1)
-    name = user.first_name
-    print(name)
-    print(doesUserHaveRole("admin"))
-    return render_template('views/admin/all_galleries.html')
+    
+    gallery_submission_querry = (Galleries
```

Comment:
This looks like a query that could go in either the GalleriesQuery as something like: get_all_galleries or in form queries as get_all_forms_group_by_galleries

Reply:
Thanks @Jessonsotoventura  Noted.

Reply:
No I think you were correct, after thinking more about where to move it, this query is a bit too specific to move into the GalleryQueries.